### ISSUE TITLE
JAWS+Chrome SVG "keyboard trap"

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#100]
This is an issue in reference to #26 which is related to the **"primitives and nested role=Menu/Menuitem"** branch. All tests use native SR navigation (arrow keys) and TAB.  

_Preface: A seasoned SR user could have the knowledge to navigate and explore the menu test webpage with more nuance, whether through manual mode switching, certain key shortcuts, etc. But this test is done by a sighted user with the SR’s default/automatic options and performed as if a new or beginner user is interacting with SVG elements. We would expect that all users could be able to navigate smoothly, regardless of experience levels._ 

<img width="803" alt="Screenshot of JAWS in a menu test environment. The focus indicator is highlighting the Help 4 checkbox." src="https://user-images.githubusercontent.com/98043076/181076013-ed780ffd-bdcf-494b-9efe-46f435e2fc61.png">

### Issue Summary
Note: This test was done using native SR arrow key navigation (selected checkbox in test.)
JAWS has a potential keyboard trap after the “Help” menu element. The user cannot continue navigating forward through content with the arrow keys. This could be due to JAWS’ automatic mode switching. The user must press TAB to continue navigating down through the page. Doing so takes them to the HTML example menu buttons and does not read the text before it. (The end of the video shows me trying to continue navigating, but I cannot.)

### Video of issue 
Example starts at 00:01:34
https://user-images.githubusercontent.com/98043076/181075112-0c83dcad-a5b4-4258-9ad2-30803fea4dd6.mp4

### Steps to reproduce
Starting at the "Interactions Instructions" heading: Use JAWS down arrow key to navigate into the SVG space with the "Enable Arrow Key Navigation" checkbox checked. Continue to press the down arrow to navigate through the menu items (the final menu item is "Help.")

### Expected behavior (Pass/Fail)
- *Pass* We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- *Pass* We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- *FAIL/Pass* We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating.
- *Pass* The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- *Pass* The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- *Pass* The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- *Pass* Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- *FAIL/Pass* We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed visually in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- *FAIL* We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- *Pass* We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).
- *Pass* We expect that a SR will be able to expand a menu element open or closed.
- *Pass* We expect that a SR will announce the expanded state correctly when navigated to.
- *Pass* We expect that a SR will announce the expanded state correctly when changed.
- *Pass* We expect that when an option has been selected, the SR will give feedback.

### Technical Details
JAWS 2022.2206.9 ILM
Chrome Version 103.0.5060.114
Windows 11 Build 22000.795
